registry/storage/shares.go

```diff
@@ -10,10 +10,11 @@ import (
 	eth2apiv1 "github.com/attestantio/go-eth2-client/api/v1"
 	"github.com/attestantio/go-eth2-client/spec/phase0"
 	spectypes "github.com/ssvlabs/ssv-spec/types"
+	"golang.org/x/exp/maps"
```

Comment:
`maps`

Reply:
yes, its part of this pr: https://github.com/ssvlabs/ssv/pull/2071